front/src/Components/CameraControls.svelte

```diff
@@ -77,8 +78,8 @@
     }
 
     let isSilent: boolean;
-    const unsubscribeIsSilent = isSilentStore.subscribe((value) => {
-        isSilent = value;
+    const unsubscribeIsSilent = availabilityStatusStore.subscribe((status) => {
+        isSilent = status === AvailabilityStatus.SILENT;
```

Comment:
Here, we should listen back to the "silentStore" (now that we added it back) and not the `availabilityStatusStore`.
Indeed, we want to display the "silent" message when the "silentStore" is true.

In the future, there might appear a new status that overrides the "silent" status. In this case, we would still want the "silent" message displayed when we are in a silent zone.